include/bluetooth/iso.h

```diff
@@ -337,49 +349,44 @@ struct bt_iso_server {
  */
 int bt_iso_server_register(struct bt_iso_server *server);
 
-/** @brief Bind ISO channels
- *
- *  Bind ISO channels with existing ACL connections, Channel objects passed
- *  (over an address of it) shouldn't be instantiated in application as
- *  standalone.
+/** @brief Creates a CIG as a central
  *
- *  @param conns Array of ACL connection objects
- *  @param num_conns Number of connection objects
- *  @param chans Array of ISO Channel objects to be created
+ *  @param[in] param     The parameters used to create and enable the CIG.
+ *                       The QOS parameters are determined by the QOS field of
+ *                       the first BIS in the BIS list of this parameter.
+ *  @param[out] out_cig  Connected Isochronous Group object on success.
  *
  *  @return 0 in case of success or negative value in case of error.
  */
-int bt_iso_chan_bind(struct bt_conn **conns, uint8_t num_conns,
-		     struct bt_iso_chan **chans);
+int bt_iso_cig_create(const struct bt_iso_cig_create_param *param,
```

Comment:
BIS -> CIS
Maybe a hint on when this can be called (I believe the idea was that it no longer requires the ACL to be created first)
Actually the HCI interface allows separate qos settings for each CIS, hence I would prefer if we keep this functionality exposed in the API and extract Qos settings from each cis_channel - maybe with fallback to the cirst cis_channel if the qos is NULL. 
Imagine a true wireless headset with microphone in only one side. Such a pair would require a unidirectional CIS for one device and a bidirectional CIS to another.
Also you could be connection to devices from two different vendors, one that supports 1 and 2 mbit, and another that supports 1mbit and coded, and want to use 2 mbit for the device that have support for it and 1 for the other.
 👍 for writing that one needs to set the QOS pointers prior to this call. It is missing details like this that makes the current API very difficult to use, as you get multiple crashes until you find the correct sequence of when pointers in the structures needs to be filled out :-)

Reply:
> Maybe a hint on when this can be called (I believe the idea was that it no longer requires the ACL to be created first)

We could make a MSC for it 

> Actually the HCI interface allows separate qos settings for each CIS, hence I would prefer if we keep this functionality exposed in the API and extract Qos settings from each cis_channel - maybe with fallback to the cirst cis_channel if the qos is NULL.

Good idea. We would need to split the QOS struct then to the CIG related values, and to the CIS related values, right? This will in turn make it harder to reuse the same struct for BIGs, but maybe not impossible.